### PR TITLE
Feat: Implement backend of Comment model and ProblemSet info/solver/commet

### PR DESCRIPTION
**Pull request by TLB-MISS**
Refactoring backend for CBV
Implement backend of Comment model
Implement backend of ProblemSet info/solver/commet

### CODE REVIEW

**Review by mman39 — APPROVED** [review]

File: backend/probloom/prob/models.py [hunk 1]
```diff
@@ -1,17 +1,85 @@
...
+    tag = CharField(max_length=100, default="default tag")
+    difficulty = SmallIntegerField(default=0)
+    content = TextField(max_length=1000, default="default content")
+    creator = ForeignKey(
+        UserStatistics, related_name="created_problem", on_delete=CASCADE
+    )
+    recommender = ManyToManyField(
+        UserStatistics, blank=True, related_name="recommended_problem"
+    )
+    solver = ManyToManyField(UserStatistics, blank=True, related_name="solved_problem")
```
Comment (mman39) [inline backend/probloom/prob/models.py:46]: Did you deleted solved model? I think it is necessary because we need result of solved problem set.

Comment (TLB-MISS) [reply backend/probloom/prob/models.py:46]: When I added the Solved model to my model, a migration error popped up so I did not add the Solved model to my model. So, it was inevitable to remove it from the test code.

**Comment by clarkindy** [comment]
Could you change the pull destination from `main` to `sprint3`?

**Comment by TLB-MISS** [comment]
> Could you change the pull destination from `main` to `sprint3`?

Oh, sorry. It's corrected.

**Review by clarkindy — APPROVED** [review]

**Review by clarkindy — APPROVED** [review]

**Pull request merged**